### DISCUSSION TITLE
Log Channel OSS Alibaba

### DISCUSSION THREAD

**Discussion by ioiofadhil** [alphasnow/aliyun-oss-laravel#33] [Q&A] [answered]
Hi. I've tried the package for filesystem and it works great! Thanks a lot! 
But I'm also wondering, can I store the log files into OSS using `Log Channel` laravel?

**Comment by alphasnow** [comment, marked as answer]
> Hi. I've tried the package for filesystem and it works great! Thanks a lot! But I'm also wondering, can I store the log files into OSS using `Log Channel` laravel?

I don't recommend using OSS as a Log channel, Because that would slow down your PHP program.
Alternatively, you can consider synchronizing log files to OSS using a scheduled task.

**Reply by ioiofadhil** [reply]
You’re right! Scheduler it is. So saving logs in local as a temporary, upload it to OSS with scheduler and also delete local log afterwards. Is that right?

**Reply by alphasnow** [reply]
Right
